api/openapi.yaml

```diff
@@ -36,138 +36,138 @@ paths:
                   items:
                     type: array
                     items:
-                      $ref: "./transactions.yaml#/components/schemas/TransactionResource"
+                      $ref: "./testsuites.yaml#/components/schemas/TestSuiteResource"
         400:
-          description: "invalid query for transactions, some data was sent in incorrect format."
+          description: "invalid query for testsuites, some data was sent in incorrect format."
         500:
-          description: "problem with getting transactions"
+          description: "problem with getting testsuites"
     post:
       tags:
         - resource-api
-      summary: "Create new transaction"
-      description: "Create new transaction"
-      operationId: createTransaction
+      summary: "Create new TestSuite"
+      description: "Create new TestSuite"
+      operationId: createTestSuite
       requestBody:
         content:
           application/json:
             schema:
-              $ref: "./transactions.yaml#/components/schemas/TransactionResource"
+              $ref: "./testsuites.yaml#/components/schemas/TestSuiteResource"
           text/yaml:
             schema:
-              $ref: "./transactions.yaml#/components/schemas/TransactionResource"
+              $ref: "./testsuites.yaml#/components/schemas/TestSuiteResource"
       responses:
         201:
           description: successful operation
           content:
             application/json:
               schema:
-                $ref: "./transactions.yaml#/components/schemas/TransactionResource"
+                $ref: "./testsuites.yaml#/components/schemas/TestSuiteResource"
             text/yaml:
               schema:
-                $ref: "./transactions.yaml#/components/schemas/TransactionResource"
+                $ref: "./testsuites.yaml#/components/schemas/TestSuiteResource"
         400:
-          description: "trying to create a transaction with an already existing ID"
+          description: "trying to create a TestSuite with an already existing ID"
         500:
-          description: "problem creating a transaction"
-  /transactions/{transactionId}:
+          description: "problem creating a TestSuite"
+  /testsuites/{testSuiteId}:
     get:
       tags:
         - resource-api
       parameters:
-        - $ref: "./parameters.yaml#/components/parameters/transactionId"
-      summary: "get transaction"
-      description: "get transaction"
-      operationId: getTransaction
+        - $ref: "./parameters.yaml#/components/parameters/testsuiteId"
```

Comment:
if we're using `TestSuite`, shouldn't the parm be `testSuitId`? looks more consistent IMO

Reply:
Makes sense, I'm going to fix it as part of the frontend PR so I can merge this already into the feature branch